src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import Image from "next/image";
+
+import FacebookIcon from "@/assets/image/icons/socialicon/icon_facebook.svg";
+import InstagramIcon from "@/assets/image/icons/socialicon/icon_instagram.svg";
+import XIcon from "@/assets/image/icons/socialicon/icon_X.svg";
+import YoutubeIcon from "@/assets/image/icons/socialicon/icon_youtube.svg";
+
+const socialIcons = [
+  { href: "https://www.facebook.com/", src: FacebookIcon, alt: "Facebook" },
+  { href: "https://www.instagram.com/", src: InstagramIcon, alt: "Instagram" },
+  { href: "https://www.youtube.com/", src: YoutubeIcon, alt: "YouTube" },
+  { href: "https://x.com/", src: XIcon, alt: "X" },
+];
+
+export default function Footer() {
+  return (
+    <footer className="w-full my-5 bg-white text-[13px] text-[color:var(--gray-400)]">
+      <div
+        className="
+          w-[90%] mx-auto
+          grid grid-cols-3 grid-rows-2
+          sm:grid-rows-1 sm:grid-cols-3
+          items-center gap-2
+          text-center sm:text-left
+        "
+      >
+        <div
+          className="
+            row-start-1 col-span-3
+            flex justify-center gap-2
+            sm:col-span-1 sm:col-start-2 sm:row-start-1
+          "
+        >
+          <a href="#" className="text-[color:var(--gray-600)]">
```

Comment:
`text-[color:var(--gray-600)]` 클래스는 `text-gray-600`으로 사용할 수 있어요! 👍

호옥시 Tailwind 관련 익스텐션이 없으시다면 아래 참고 해주세요! 😆
훨씬 작업하기 편해요!
[Tailwind CSS IntelliSense Extension](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss)

```suggestion
          <a href="#" className="text-[color:var(--gray-600)]">
```